modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -391,52 +391,32 @@
 	adjustmask(user)
 
 // Donation reward for TheOOZ
-/obj/item/clothing/mask/kindle
-	name = "\proper The mask of Kindle"
-	desc = "The mask which belongs to Nanotrasen's Outpost Captain Kindle, it is the symbol of her 'Kindled' cult. The material feels like it's made entirely out of inexpensive plastic."
+/obj/item/clothing/mask/animal/kindle
+	name = "wolf mask"
+	desc = "A dark mask in the shape of a wolf's head.<br>The material feels like it's made entirely out of inexpensive plastic."
```

Comment:
using a macro like `\n` would be preferable to using straight html

Reply:
this didnt work and idunno why 😔